packages/plugin/src/theia.d.ts

```diff
@@ -12084,6 +12084,11 @@ export module '@theia/plugin' {
          * Label will be rendered next to authorName if exists.
          */
         label?: string;
+
+        /**
+         * Optional timestamp.
+         */
+        timeStamp?: Date;
```

Comment:
[VS Code uses `timestamp`](https://github.com/microsoft/vscode/blob/2c8b54bbbf0167e52ef4602a9ee14e1223fb0d7c/src/vscode-dts/vscode.d.ts#L15384). Can Theia use the same?

`timeStamp` -> `timestamp`?

Reply:
Right, @rschnekenbu if you run the [vscode-theia-comparator](https://github.com/eclipse-theia/vscode-theia-comparator) against this branch you'll see that `timestamp` is still marked as **unsupported**:

![image](https://user-images.githubusercontent.com/40359487/208461914-4a86d80b-f555-4dac-b1de-b20727da4998.png)

Reply:
the timestamp typo has been fixed, as all last comments